package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "pretender",
   "version": "2.1.0",
-  "main": "./pretender.js",
+  "main": "./dist/pretender.js",
+  "module": "./src/pretender.es.js",
   "description": "Pretender is a mock server library for XMLHttpRequest and Fetch, that comes with an express/sinatra style syntax for defining routes and their handlers.",
   "license": "MIT",
   "engines": {
     "node": ">=4"
   },
   "scripts": {
+    "prepublishOnly": "npm run build && npm run tests-only",
```

Comment:
Making sure that the build step is ran before publishing a new version, in case a PR bypassed a red CI somehow.